_dockerize/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:10
+
+# Create app directory
+WORKDIR /usr/src/app
+
+# Install app dependencies
+# A wildcard is used to ensure both package.json AND package-lock.json are copied
+# where available (npm@5+)
+COPY *.json ./
+
+RUN npm install express --save
+RUN npm install --save-dev typescript @types/node eslint eslint-plugin-import nodemon ts-node
```

Comment:
Since you copy the packages.json, you don't need to reinstall the express or any other dependencies. All you need to do is:
`npm install --production` for dependencies 
or
`npm install` for dependencies and devDependencies

https://nodejs.dev/npm-dependencies-and-devdependencies